packages/cloud-cognitive/src/components/Datagrid/styles/useExpandedRow.scss

```
@@ -0,0 +1,13 @@
+/*
+* Licensed Materials - Property of IBM
+* 5724-Q36
+* (c) Copyright IBM Corp. 2020 - 2021
+* US Government Users Restricted Rights - Use, duplication or disclosure
+* restricted by GSA ADP Schedule Contract with IBM Corp.
+*/
+
+@import './variables';
+
+.#{$block-class}__expanded_row_content {
```

```suggestion
.#{$block-class}__expanded-row-content {
```

We follow BEM (`.block__element--modifier`) conventions and in this case, we‘d typically use hyphens than underscores here.